packages/RNTester/js/examples/Alert/AlertExample.js

```diff
@@ -152,7 +184,57 @@ const styles = StyleSheet.create({
   },
 });
 
-module.exports = {
-  AlertExample,
-  SimpleAlertExampleBlock,
-};
+exports.title = 'Alerts';
+exports.description =
+  'Alerts display a concise and informative message ' +
+  'and prompt the user to make a decision.';
+exports.examples = [
+  {
+    title: 'Alert with default Button',
+    description:
+      "It can be used to show some information to user that doesn't require an action.",
```

Comment:
the rest of the file follows `single quote` styling, should we change this one too ?

Reply:
The double quote is needed because there is a single quote `'` inside the string.